routing/routing_tests/checkpoint_predictor_test.cpp

```diff
@@ -0,0 +1,145 @@
+#include "testing/testing.hpp"
+
+#include "routing/checkpoint_predictor.hpp"
+
+#include "geometry/point2d.hpp"
+
+#include <cstdint>
+#include <utility>
+#include <vector>
+
+using namespace std;
+
+namespace
+{
+void TestAlmostEqual(double v1, double v2)
+{
+  double constexpr kEpsMeters = 1.0;
+  TEST(my::AlmostEqualAbs(v1, v2, kEpsMeters), (v1, v2));
```

Comment:
TEST expressions must be inside UNIT_TEST or UNIT_CLASS_TEST. Otherwise, use CHECKs.